src/index.js

```diff
@@ -11,25 +11,32 @@ import io from "socket.io-client";
 import reportWebVitals from "./reportWebVitals";
 
 const startPos = 50;
+const starDirection = Math.random() < 0.5;
```

Comment:
star?

Reply:
Maybe make a function out of this because this way the same random number will be used for x and y directions all the time which decreases 'randomness'